modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/experiment/ExperimentTest.java

```diff
@@ -241,56 +228,31 @@ public void t_createAndValidateExperiment() throws java.text.ParseException {
     public Object[][] badExperimentsPOST() {
         Experiment experiment = new Experiment().setDescription("Sample hypothesis.");
         return new Object[][]{
-                // FIXME: jwtodd
-//                new Object[] { new Experiment(experiment.setSamplingPercent(completeExperiment.samplingPercent)), "Invalid input", HttpStatus.SC_BAD_REQUEST },
                 new Object[]{new Experiment(experiment.setSamplingPercent(completeExperiment.samplingPercent)), "Experiment application name cannot be null or an empty string", HttpStatus.SC_BAD_REQUEST},
-                // FIXME: jwtodd
-//                new Object[] { new Experiment(experiment.setStartTime(completeExperiment.startTime)), "Invalid input", HttpStatus.SC_BAD_REQUEST },
                 new Object[]{new Experiment(experiment.setStartTime(completeExperiment.startTime)), "Experiment application name cannot be null or an empty string", HttpStatus.SC_BAD_REQUEST},
-//                new Object[] { new Experiment(experiment.setEndTime(completeExperiment.endTime)), "Invalid input", HttpStatus.SC_BAD_REQUEST },
                 new Object[]{new Experiment(experiment.setEndTime(completeExperiment.endTime)), "Experiment application name cannot be null or an empty string", HttpStatus.SC_BAD_REQUEST},
-//                new Object[] { new Experiment(experiment.setLabel(completeExperiment.label)), "Invalid input", HttpStatus.SC_BAD_REQUEST },
                 new Object[]{new Experiment(experiment.setLabel(completeExperiment.label)), "Experiment application name cannot be null or an empty string", HttpStatus.SC_BAD_REQUEST},
-                // FIXME: jwtodd
-//                new Object[] { new Experiment(experiment.setApplication(ApplicationFactory.defaultApplication())), "Uniqueness constraint violated", HttpStatus.SC_BAD_REQUEST },
-                new Object[]{new Experiment(experiment.setApplication(ApplicationFactory.defaultApplication())), "An unique constraint was violated: An active experiment with label \"SW50ZWdyVGVzdA_1461232889078App_PRIMARY\".\"SW50ZWdyVGVzdA_Experiment_14612328892453\" already exists (id = 70139a10-489b-49bd-ac4c-c7c92ec79917) (null)", HttpStatus.SC_BAD_REQUEST},
-
-                // FIXME: jwtodd
-//                new Object[] { ExperimentFactory.createExperiment().setState(Constants.EXPERIMENT_STATE_DRAFT), "Invalid input", HttpStatus.SC_BAD_REQUEST },
+                new Object[]{new Experiment(experiment.setApplication(ApplicationFactory.defaultApplication())), "An unique constraint was violated: ", HttpStatus.SC_BAD_REQUEST},
```

Comment:
This test sample is not working for me - I get an Assertion error for the response code since it is 201 and not 400. Two other tests are not working since they expect 400 but get 404. I would like to point you to those as well but it hard to find them from the unitTests output in the Dataprovider :/